securesignals-ima-devapp/src/androidTest/java/com/uid2/dev/ExampleInstrumentedTest.kt

```diff
@@ -0,0 +1,24 @@
+package com.uid2.dev
+
+import androidx.test.platform.app.InstrumentationRegistry
+import androidx.test.ext.junit.runners.AndroidJUnit4
+
+import org.junit.Test
+import org.junit.runner.RunWith
+
+import org.junit.Assert.*
+
+/**
+ * Instrumented test, which will execute on an Android device.
+ *
+ * See [testing documentation](http://d.android.com/tools/testing).
+ */
+@RunWith(AndroidJUnit4::class)
+class ExampleInstrumentedTest {
```

Comment:
Example Android test added by Android Studio. It can be removed (along with dependencies).

Reply:
Done.